### ISSUE TITLE
Hovercards missing from Chrome web store

### ISSUE THREAD

**Issue by arcdelsol** [kogg/hovercards#290]
Did this extension get deleted from the Chrome Web store?
The link is returning 404

**Comment by daylenca** [comment]
I'm getting the same error message too 😢

**Comment by itsbrex** [comment]
Same here! Fack!

**Comment by saiichihashimoto** [comment]
Truth is, we've all moved on and hovercards hasn't gotten any development in quite a long time. It's build is broken and we didn't find it worth it to renew the chrome extension certificate as well.

**Comment by itsbrex** [comment]
All good. Found basically another extension that essentially does the same thing. For anyone else interested, check out URL Render: 

https://chrome.google.com/webstore/detail/url-render/flhclpkhoiajoikkabbfbinnjapaflog

**Comment by saiichihashimoto** [comment]
Yeah, sorry about that! The code for hovercards is also available, so anyone willing to make a PR to fix it would also be appreciated!
